pycbc/detector.py

```diff
@@ -444,3 +445,138 @@ def time_delay_from_earth_center(self, right_ascension, declination, t_gps):
             np.array([np.float32(earth.x), np.float32(earth.y),
                       np.float32(earth.z)]), right_ascension,
             declination, t_gps)
+
+    def interpolate_hij(self, hp, hc):
```

Comment:
This isn't really in hij form, should probably retitle to interpolate_waveform, or similar.

Reply:
changed to `interpolate_waveform`